load_balancer.tf

```diff
@@ -1,8 +1,11 @@
 #Load Balancer
 
-resource "aws_elb" "reutlb" {
-  name               = "reutlb"
-  availability_zones = var.azs
+resource "aws_elb" "nginx" {
+  name      = "nginx-elb"
+  subnets   = [aws_subnet.public_subnet[0].id, aws_subnet.public_subnet[1].id]
+  instances = [aws_instance.nginx[0].id, aws_instance.nginx[1].id]
+  security_groups = [aws_security_group.nginx_instnaces_access.id]
+
   listener {
```

Comment:
- I renamed the resource.
- Instead of using `availability_zones` I used `subnets`. More simple to use. And the subnets are already in different AZ.
- The elb will not work if you do not assign it the desired instances :) I added `instances` value and set our two nginx instances to it. `[aws_instance.nginx[0].id, aws_instance.nginx[1].id]`
- Our elb also needs sg with rules who can access to it. I attached it the sg of our Nginx instances.